README.md

```diff
@@ -14,12 +14,29 @@ It also contains scripts for local development of Mobile Services (using `Minish
 
 1. Make sure you are targeting OpenShift instance with installed Ansible Service Broker (run `oc projects` and search for `openshift-automation-service-broker` or `openshift-ansible-service-broker`)
 2. Run the installation playbook:
+    
+    If you want to use the community releases, run the following command:
 
-```
-ansible-playbook install-mobile-services.yml
-```
+    ```
+    ansible-playbook install-mobile-services.yml
+    ```
+
+    If you want to use the productized releases from Red Hat Container Catalog, please use the following command:
+
+    ```
+    ansible-playbook install-mobile-services.yml -e "ansible_playbookbundle_registry_type=rhcc"
+    ```
+
+    If authentication is required, you can set them via these variables: `rhcc_registry_auth_name` and `rhcc_registry_auth_type`.
```

Comment:
We should refer to some documentation where this variables are explained, e.g. [here](https://github.com/openshift/ansible-service-broker/blob/master/docs/config.md#registry-configuration)